static/js/data_handler.js

```diff
@@ -74,6 +75,46 @@ export let dataHandler = {
     },
     createNewCard: function (cardTitle, boardId, statusId, callback) {
         // creates new card, saves it and calls the callback function with its data
-    }
+    },
     // here comes more features
+    submitNewTitle: function submit_new_title(event) {
+                        let header = event.currentTarget.parentElement;
+                        let newTitle = header.querySelector(".new-board-title");
+                        console.log(newTitle.value);
+                        if (newTitle.value === ''){
+                            newTitle.value = newTitle.getAttribute("value");
+                        }
+                        let saveButton = header.querySelector(".title-save-button");
+                        header.removeChild(newTitle);
+                        header.removeChild(saveButton);
+                        let boardTitle = document.createElement('span');
```

Comment:
When you are not going to write down a line like `boardTitle = something` in the future again, consider using `const` keyword instead of `let`.